docs/setup/configuring-reporting.asciidoc

```diff
@@ -41,11 +41,16 @@ To troubleshoot the problem, start the {kib} server with environment variables t
 [float]
 [[grant-user-access]]
 === Grant users access to reporting
+When security is enabled, granting users privilege to generate reports can be controlled by <<kibana-privileges, {kib} application privileges>>. This lets you create custom roles that control the spaces and applications where users are allowed to generate reports.
 
-When security is enabled, access to the {report-features} is controlled by roles and <<kibana-privileges, privileges>>. With privileges, you can define custom roles that grant *Reporting* privileges as sub-features of {kib} applications. To grant users permission to generate reports and view their reports in *Reporting*, create and assign the reporting role.
-
-[[reporting-app-users]]
-NOTE: In 7.12.0 and earlier, you grant access to the {report-features} by assigning users the `reporting_user` role in {es}. 
+. Enable **application privileges** in Reporting. To do this, you must turn off the default user access control features in `kibana.yml` with:
++
+[source,yaml]
+------------------------------------
+xpack.reporting.roles.enabled: false
+------------------------------------
++
+NOTE: If you keep the default settings, you can still create a custom role that grants reporting privilege: the default role is `reporting_user`. This behavior is being deprecated as it does not allow application-level access control for {report-features}, and it does not allow API keys or authentication tokens to authorize report generation. See <<reporting-advanced-settings, reporting security settings>> for information and caveats about the deprecated access control features.
```

Comment:
I think this is really great. Mentioning API keys is also really useful, do you think we can give an indication of the benefits of using API keys for report generation? Perhaps we could link out to that area of the docs. Let me know what you think!

Reply:
```suggestion
NOTE: If you use the default settings, you can still create a custom role that grants reporting privileges. The default role is `reporting_user`. This behavior is being deprecated and does not allow application-level access controls for {report-features}, and does not allow API keys or authentication tokens to authorize report generation. Refer to <<reporting-advanced-settings, reporting security settings>> for information and caveats about the deprecated access control features.
```

Reply:
> do you think we can give an indication of the benefits of using API keys for report generation? 

A big downside to adding text about this is, Watcher does not support API Key authentication for the reporting email action, and a lot of users depend on it:  https://www.elastic.co/guide/en/kibana/7.14/automating-report-generation.html#use-watcher

At some point in the future when automating report generation uses Kibana APIs, there will be more interest for using API Keys for report generation.